common/c_cpp/src/c/mempool.c

```diff
@@ -76,6 +76,7 @@ memoryNode*
 memoryPool_addNode (memoryPool* pool, size_t nodeSize)
```

Comment:
As discussed off-ticket, there is not in your version of Visual Studio. With that in mind, let's just keep these changes in your fork and I'll work out any conflicts when we merge entitlements too.